new/vcloud-manager-blueprint.yaml

```diff
@@ -0,0 +1,867 @@
+          destination_port: 5672
+          action: allow
+          description: >
+            AQMP
+          protocol: TCP
+        - source: external
+          destination: internal
+          destination_port: 8100
+          action: allow
+          description: >
+            Diamond
+          protocol: TCP
+        - source: external
+          destination: internal
+          destination_port: 8101
+          action: allow
+          description: >
+            Diamond
+          protocol: TCP
+        - source: internal
+          destination: any
+          action: allow
+          description: >
+            backward network connection for host updates
+          protocol: any
+      vcloud_config: { get_property: [vcloud_configuration, vcloud_config] }
+
+  manager_public_ip:
+    type: cloudify.vcloud.nodes.PublicNAT
+    properties:
+      nat:
+        edge_gateway: { get_input: edge_gateway }
+        public_ip: { get_input: public_ip }
+      rules:
+        - type: DNAT
+          protocol: tcp
+          original_port:  { get_input: ssh_port }
+          translated_port: 22
+        - type: DNAT
+          protocol: tcp
+          original_port: 80
+          translated_port: 80
+        - type: DNAT
+          protocol: tcp
+          original_port: 443
+          translated_port: 443
+      vcloud_config: { get_property: [vcloud_configuration, vcloud_config] }
+
+  management_port:
+    type: cloudify.vcloud.nodes.Port
+    properties:
+      port:
+        network: { get_input: management_network_name }
+        ip_allocation_mode: { get_input: management_port_ip_allocation_mode }
+        ip_address: { get_input: management_port_ip_address }
+        primary_interface: true
+      vcloud_config: { get_property: [vcloud_configuration, vcloud_config] }
+    relationships:
+      - target: management_network
+        type: cloudify.vcloud.port_connected_to_network
+
+  manager_host_server:
+    type: cloudify.vcloud.nodes.Server
+    properties:
+      server:
+        name: { get_input: server_name }
+        catalog: { get_input: catalog }
+        template: { get_input: template }
+        hardware:
+          cpu: 3
+          memory: 4096
+        guest_customization:
+          computer_name: { get_input: server_name }
+      install_agent: false
+      management_network: { get_input: management_network_name }
+      vcloud_config: { get_property: [vcloud_configuration, vcloud_config] }
+    relationships:
+      #- target: management_network
+      #  type: cloudify.vcloud.server_connected_to_network
+      - target: manager_public_ip
+        type: cloudify.vcloud.server_connected_to_public_nat
+      - target: ssh_keypair
+        type: cloudify.vcloud.server_connected_to_keypair
+      - target: management_port
+        type: cloudify.vcloud.server_connected_to_port
+      - target: manager_security_group
+        type: cloudify.vcloud.server_connected_to_security_group
+
+  manager_host:
```

Comment:
Well, manager_configuration is meant to be a node in which we dump configuration and runtime properties for others to read and use, so we probably don't want to merge them.

What I meant is, that we can merge manager_host_server and manager_host (like in the openstack manager blueprint). I fail to understand why we need both.